dap4/dap4.xsd

```diff
@@ -1,199 +1,149 @@
-    </xs:complexType>
-  </xs:element>
-  <xs:complexType name="variabledef">
-    <xs:choice minOccurs="0" maxOccurs="unbounded">
-      <xs:element ref="dap:Dim"/>
-      <xs:element ref="dap:Map"/>
-      <xs:element ref="dap:metadata"/>
-    </xs:choice>
-    <xs:attribute name="name" use="required" type="xs:string"/>
-  </xs:complexType>
-  <xs:element name="Map">
-    <xs:complexType>
-      <xs:attribute name="name" use="required" type="xs:string"/>
-    </xs:complexType>
-  </xs:element>
-  <xs:element name="Structure" substitutionGroup="dap:variable">
-    <xs:complexType>
-      <xs:choice minOccurs="0" maxOccurs="unbounded">
-        <xs:element ref="dap:Dim"/>
-        <xs:element ref="dap:variable"/>
-        <xs:element ref="dap:metadata"/>
-      </xs:choice>
-      <xs:attribute name="name" use="required" type="xs:string"/>
-    </xs:complexType>
-  </xs:element>
-  <xs:complexType name="metadatalist">
-    <xs:sequence>
-      <xs:element minOccurs="0" maxOccurs="unbounded" ref="dap:metadata"/>
-    </xs:sequence>
-  </xs:complexType>
-  <xs:element name="metadata" abstract="true">
-    <xs:complexType>
-      <xs:sequence>
-        <xs:group minOccurs="0" ref="dap:namespace"/>
-        <xs:choice minOccurs="0" maxOccurs="unbounded">
-          <xs:element name="Value">
-            <xs:complexType>
-              <xs:attribute name="value" use="required">
-                <xs:simpleType>
-                  <xs:union memberTypes="xs:string xs:string xs:string xs:string xs:string xs:string"/>
-                </xs:simpleType>
-              </xs:attribute>
-            </xs:complexType>
-          </xs:element>
-          <xs:element name="Value" type="xs:string"/>
+<xs:schema targetNamespace="http://xml.opendap.org/ns/DAP/4.0#"
+    xmlns="http://xml.opendap.org/ns/DAP/4.0#"
+    xmlns:dap="http://xml.opendap.org/ns/DAP/4.0#"
+    xmlns:xml="http://www.w3.org/XML/1998/namespace"
+    xmlns:xlink="http://www.w3.org/1999/xlink"
+    xmlns:dc="http://purl.org/dc/elements/1.1/" 
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.w3.org/2001/XMLSchema http://www.w3.org/2001/XMLSchema.xsd"
+    elementFormDefault="qualified" 
+    attributeFormDefault="unqualified" 
+    xml:lang="en">
+
+    <!-- Note that these schemas are local; the XML one can be found at 
+        http://www.w3.org/2001/xml.xsd. While there is no official schema for 
+        xlink a \'non-normative\' one is provided in the appendix of
+        http://www.w3.org/TR/xlink11/. jhrg 2/7/12 -->
+
+    <xs:import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="xml.xsd"/>
+    <!-- http://www.w3.org/2001/xml.xsd -->
+    <xs:import namespace="http://www.w3.org/1999/xlink" schemaLocation="xlink.xsd"/>
+    <!-- Copied from the appendix of http://www.w3.org/1999/xlink -->
+    <xs:import namespace="http://purl.org/dc/elements/1.1/" schemaLocation="dc.xsd"/>
```

Comment:
One idea about refactoring came up with this: This dublincore (dc.xsd) schema, apparently used for dates (time), is used by the complexType=BlobType (see line 480). `BlobType` does NOT show up in the dap4 specification, and seems that it was an early prototype Type for dap4 and not used in DAP4. 

Should this be removed/refactored?